frontend/packages/knative-plugin/src/components/add/event-sources/__tests__/EventSourcesSelector.spec.tsx

```diff
@@ -27,6 +28,38 @@ describe('EventSourcesSelector', () => {
     wrapper = shallow(<EventSourcesSelector eventSourceList={eventSourceList} />);
   });
 
+  it('should not render FormSection if no more than one eventSource present', () => {
+    const eventSourceList = {
+      SinkBinding: {
+        title: 'sinkBinding',
+        iconUrl: 'sinkBindingIcon',
+        name: 'SinkBinding',
+        displayName: 'Sink Binding',
+      },
+    };
+    wrapper = shallow(<EventSourcesSelector eventSourceList={eventSourceList} />);
+    expect(wrapper.find(FormSection)).toHaveLength(0);
```

Comment:
This can be written like:
```suggestion
    expect(wrapper.find(FormSection).exists()).toBe(false);
```

Reply:
done